plugins/imfile/imfile.c

```diff
@@ -2892,6 +2927,8 @@ CODEmodInit_QueryRegCFSLineHdlr
 	  	NULL, &cs.maxLinesAtOnce, STD_LOADABLE_MODULE_ID));
 	CHKiRet(omsdRegCFSLineHdlr((uchar *)"inputfiletrimlineoverbytes", 0, eCmdHdlrSize,
 	  	NULL, &cs.trimLineOverBytes, STD_LOADABLE_MODULE_ID));
+	CHKiRet(omsdRegCFSLineHdlr((uchar *)"inputfileignoreolderthan", 0, eCmdHdlrSize,
```

Comment:
please remove support for the old config system - we do NOT provide new features via the old system.